test/leader/README.md

```diff
@@ -0,0 +1,4 @@
+# Leader Tests
+## Prerequisites
+1. Leader vehicle must be powered on during all testing. 
+2. Controller must be connected to vehicle for the duration of testing in order to give input.
```

Comment:
Controller.

The controller should not be giving any input.

Reply:
modified to driver gives input